ta/os_test/os_test.c

```diff
@@ -1298,3 +1299,101 @@ TEE_Result ta_entry_client_identity(uint32_t param_types, TEE_Param params[4])
 
 	return res;
 }
+
+__thread int os_test_tls_a;
+__thread int os_test_tls_b = 42;
+
+TEE_Result ta_entry_tls_test_main(void)
+{
+        if (os_test_tls_a != 0) {
+		EMSG("os_test_tls_a=%d, expected 0", os_test_tls_a);
+                return TEE_ERROR_GENERIC;
```

Comment:
fix indentation, here and below and the 3 other occurrences below.

Reply:
ok